### DISCUSSION TITLE
Add a ci to check typo

### DISCUSSION THREAD

**Discussion by yuxqiu** [TrueLearnAI/truelearn#42] [General]
@KD-7 I think we can add a ci to check for typos. Please see [typos-cli](https://github.com/crate-ci/typos).

It seems that the only thing we need to is to use its Github Action with `files: truelearn`.

**Comment by KD-7** [comment]
> @KD-7 I think we can add a ci to check for typos. Please see [typos-cli](https://github.com/crate-ci/typos).
> 
> It seems that the only thing we need to is to use its Github Action with `files: truelearn`.

Sounds good to me , make sure you checkout the repo in the first step
